### ISSUE TITLE
Werebat warrior missing affliction connection to strike

### ISSUE THREAD

**Issue by dharkus5000** [foundryvtt/pf2e#19287]
the Jaws strike (which is linked to change shape and only present when not in humanoid form) should be connected to the curse of the werebat, so it shows up when you attack with that strike in the chat card, but it's not
